### DISCUSSION TITLE
Update Sonoff RF Bridge 433 Documentation

### DISCUSSION THREAD

**Discussion by sneakygecko** [arendst/Tasmota#12883] [Ideas]
Hi,

On the help page for the RF Bridge
https://tasmota.github.io/docs/devices/Sonoff-RF-Bridge-433/

There is a section which states:

'''
Hardware Preparation

This section applies only to the R2 revision of the Sonoff RF Bridge

Apply this modification only if you intend to use USB for powering the device during flash process... then goes on to show where to cut USB data lines
'''

This section should be below Flashing the firmware as instructions read

'''
5. Power up the bridge via the 3.3V & GND pins. (See below if you need to power the device through the USB power port)
'''

Also in point four I had my switch in the ON position while flashing
'''
4. Switch the ON/OFF switch on the board to 'OFF'.
'''

Thanks, 
You guys rock.

**Comment by ascillato** [comment]
Hi,

Please, help us and do the modifications. You can edit the page by clicking on the pencil that is at the top right of the page. Thanks.

![image](https://user-images.githubusercontent.com/35405447/129461440-b4826458-1c0c-4ff8-8c22-64504afaffb3.png)

**Comment by sneakygecko** [comment]
Done & made Pull request :)

**Reply by ascillato** [reply]
Thanks !!!
